src/app.js

```diff
@@ -30,6 +74,18 @@
 //3. Counsel her daughter herself
 //4. Lock her daughter in her room
 //Given a value, return which of these above actions Lily would take
+function aDifficultChoice(choice){
+    switch (choice) {
+        case 1:
+            return "Take her daughter to a doctor";
+        case -1:
+            return "Break down and give up all hope";
+        case undefined:
+            return "Wasn't able to decide";
+        case 'I give up':
+            return "Refused to do anything for Karen";
```

Comment:
Where is the default case?
Is it mandatory or It will be useful for exception handling

Reply:
ok .I will apply